src/views/opm.tsx

```diff
@@ -479,149 +479,153 @@ const OPM: React.FC = () => {
-                            value={form.value}
-                            maxDate={
-                              form.name === "date" ? CURRENT_PST_DATE : null
-                            }
-                            dateFormat="dd-MM-yyyy hh:mm"
-                          />
-                        )}
-                        {form.type === INPUT_TYPES.dropdown && (
-                          <CustomDropdown
-                            value={form.value}
-                            name={form.name}
-                            onChange={(e) => handleFormChange(e)}
-                            containerclassname="sm:w-20w lg:!w-11w"
-                            imageclassname="relative left-25 z-1"
-                            dropdownIcon={<CustomImage src={ArrowDownIcon} />}
-                            icon={form.icon}
-                            options={form.options}
-                            label={form.label}
-                            optionLabel="name"
-                            placeholder=""
-                          />
-                        )}
-                      </div>
-                    );
-                  })}
-                </form>
+                <form id="custom-hover" className="lg:flex sm:gap-4 opmFilters  sm:grid sm:grid-cols-3  sm:mb-4">
+                {formFields.map((form, index) => {
+                  return (
+                    <div
+                      className="flex flex-1"
+                      key={index}
+                    >
+                      {form.type === INPUT_TYPES.text && (
+                        <CustomInputText
+                          value={form.value}
+                          name={form.name}
+                          label={form.label}
+                          icon={form.imgsrc}
+                          placeholder={form.label}
+                          className="h-39"
+                          imageclassname="!top-[1.9rem]"
+                          containerClassName="lg:max-w-[8rem]"
+                          onChange={(event) => handleFormChange(event)}
+                        />
+                      )}
+                      {form.type === INPUT_TYPES.time && (
+                        <CustomCalendar
+                          name={form.name}
+                          titleclassname="top-1.25r"
+                          containerclassname="lg:min-w-11r"
+                          imageclassname="h-5 w-5 relative top-1.75r left-0.5w z-1"
+                          placeholder={DATE_AND_TIME_FORMATS.MM_DD_YYYY_HH_MM}
+                          title={form.label}
+                          showTime={form.showTime}
+                          iconPos={form.iconPos || "left"}
+                          imgsrc={form.imgsrc}
+                          onChange={(event) => handleFormChange(event)}
+                          value={form.value}
+                          maxDate={form.name === "date" ? new Date() : null}
```

Comment:
I think this should be current PST. Please check with Thasnim

Reply:
maxDate={ form.name === "date" ? CURRENT_PST_DATE : null}